roles/db-copy/tasks/active-copy.yml

```diff
@@ -13,212 +13,272 @@
 # limitations under the License.
 
 ---
-- name: Active-copy | Check pmon process
+- name: Active-copy | Check if standby database is already running
   shell: |
-    set -o pipefail
-    ps -ef | ( grep pmon || true ) | ( grep -i {{ oracle_sid }}$ || true ) | ( grep -v grep || true ) | wc -l
-  changed_when: false
-  register: pmon_proc
-  tags: active-duplicate
-
-- name: Active-copy | Add oratab entry
-  lineinfile:
-    path: /etc/oratab
-    regexp: '^{{ oracle_sid }}\:'
-    line: "{{ oracle_sid }}:{{ oracle_home }}:N"
-    owner: "{{ oracle_user }}"
-    group: "{{ oracle_group }}"
-  become: true
-  tags: active-duplicate
-
-- name: Active-copy | Set random password
-  set_fact:
-    sys_pass: "{{ lookup('password', '/dev/null length=16 chars=ascii_letters,digits') }}0#_"
-  tags: active-duplicate
-
-- name: Active-copy | Get primary db password file information
-  shell: |
-    set -o pipefail
-    srvctl config db -d {{ db_name }} | grep "^Password file"
+    srvctl status database -d {{ standby_name }} || true
```

Comment:
We plan to add support for filesystem configurations fairly soon, including data guard;  will these changes make it more difficult?  

[oracle@oracle-linux-94 ~]$ srvctl status database -d test
****Unable to retrieve Oracle Clusterware home.
Start Oracle Clusterware stack and try again.

Reply:
I guess we're already relying on srvctl anyway (just adding it later), so we might as well leverage it.  One follow-up question though:  If we do a "|| true" and presumably continue unconditionally, do we get a benefit from checking the status at all here?